16x9/DialogFullScreenInfo.xml

```diff
@@ -698,64 +702,71 @@
 		<!-- LiveTV Plot-->
 		<control type="group" id="9006">
 			<top>10</top>
-			<animation effect="slide" end="0,100" time="240" tween="quadratic" condition="Window.IsActive(videoosd)">Conditional</animation>
-			<animation effect="fade" start="0" end="100" time="400">WindowOpen</animation>
-			<animation effect="fade" start="100" end="0" time="400">WindowClose</animation>
```

Comment:
The fade-animation is working only the first time I press "Info". If I press "Info" again to close it and open it again with "Info" there is no animation.
Also I can't get any slide animation to work. And let me say that I like the fade animation more than a sliding one.

There shouldn't be two places for Live-TV animations, atm Includes_LiveTV and DialogFullscreenInfo. Because of this, I like includes ;)